google/cloud/logging_v2/handlers/handlers.py

```diff
@@ -92,15 +93,19 @@ def filter(self, record):
         record._span_id = getattr(record, "span_id", inferred_span) or None
         record._http_request = getattr(record, "http_request", inferred_http)
         record._source_location = CloudLoggingFilter._infer_source_location(record)
-        record._labels = {**self.default_labels, **user_labels} or None
+        # add logger name as a label if possible
+        logger_label = {"python_logger": record.name} if record.name else {}
```

Comment:
Mostly for backwards compatibility. Previously, all logs were json logs with a `message` field for user input and `python_logger` with the logger name as the other field.

I wanted to give users control over the entire payload so they could log arbitrary json objects, but I thought I should still put `python_logger` as a label in case some users were relying on it